VisualPinball.Unity/VisualPinball.Unity.Editor/Import/VpxImportWizardSettings.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace VisualPinball.Unity.Editor.Import
+{
+	[Serializable]
+	public class VpxImportWizardSettings : MonoBehaviour
+	{
+		public static bool ApplyPatch
+		{
+			get => EditorPrefs.GetBool("ApplyPatch", false);
+			set => EditorPrefs.SetBool("ApplyPatch", value);
+		}
+
+		public static string VpxPath
+		{
+			get => EditorPrefs.GetString("VpxPath", "");
+			set => EditorPrefs.SetString("VpxPath", value);
+		}
+
+		public static string TableName
+		{
+			get => EditorPrefs.GetString("TableName", "");

```

Comment:
Can we set the default to `%TABLENAME%` here so it defaults to the standard import behavior?

Reply:
The default (blank) reverts to this, but it's for sure better more verbose if %TABLENAME% is used. I only added it in case someone wants to have that table name in addition to some text like "Pinball [Table1]" for "Pinball [%TABLENAME%]"